src/Mgate/PersonneBundle/Entity/Personne.php

```diff
@@ -142,6 +148,29 @@ public function getPoste()
         }
     }
 
+    /** GDPR implementation, erase all personnal data */
```

Comment:
I think a:

```php
/**
 * {@inheritdoc}
 */
```

block is more appropriate here

Reply:
Sure.